arch/arm/soc/st_stm32/stm32f0/Kconfig.defconfig.stm32f072xb

```diff
@@ -0,0 +1,22 @@
+# Kconfig - ST Microelectronics STM32F072XB MCU
+#
+# Copyright (c) 2017 BayLibre, SAS
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_STM32F072XB
+
+config SOC
+	string
+	default stm32f072xb
+
+config FLASH_PAGE_SIZE
```

Comment:
I haven't looked at the F0 flash driver yet, but if this value isn't coming from DT, why not just use FLASH_PAGE_SIZE as defined in drivers/include/stm32f0xx_hal_flash_ex.h?